sdk/openai/openai/review/openai-types.api.md

```diff
@@ -200,19 +321,47 @@ export interface OnYourDataConnectionStringAuthenticationOptions extends OnYourD
     type: "connection_string";
 }
 
+// @public
+export type OnYourDataContextProperty = "citations" | "intent" | "all_retrieved_documents";
+
 // @public
 export interface OnYourDataDeploymentNameVectorizationSource extends OnYourDataVectorizationSourceParent {
     deployment_name: string;
+    dimensions?: number;
     type: "deployment_name";
 }
 
+// @public
+export interface OnYourDataEncodedApiKeyAuthenticationOptions extends OnYourDataAuthenticationOptionsParent {
+    encoded_api_key: string;
+    type: "encoded_api_key";
+}
+
 // @public
 export interface OnYourDataEndpointVectorizationSource extends OnYourDataVectorizationSourceParent {
-    authentication: OnYourDataApiKeyAuthenticationOptions;
+    authentication: OnYourDataVectorSearchAuthenticationOptions;
```

Comment:
I believe this should be APIKey and AccessKey [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-10-01-preview/inference.yaml#L3704-L3707)